src/univalue/univalue.cpp

```diff
@@ -175,21 +182,21 @@ bool UniValue::checkObject(const std::map<std::string,UniValue::VType>& t)
 const UniValue& UniValue::operator[](const std::string& key) const
 {
     if (typ != VOBJ)
-        return nullValue;
+        return NullUniValue;
 
     int index = findKey(key);
```

Comment:
Ugh, maybe use ``ssize_t``? But then we need to prevent the length of ``values`` exceeding ``SSIZE_MAX``.